### DISCUSSION TITLE
Removing Gitbook Stamp at the Bottom Right

### DISCUSSION THREAD

**Discussion by yankikahn** [GitbookIO/community#349] [0. General bugs and questions]
I've seen one other discussion that was similar, but because of my eagerness to make a decision, I wanted to post a more specific question to verify 🤔.

We are currently in the process of determining the platform to host our support center and help documentation 📋. I saw in a similar conversation that there is no way to remove the GitBook watermark. However, I find that very hard to believe 🧐. I see on a GitBook built website (docs.cascade.io), that there is no GitBook watermark. Can you please help me understand why it can't be removed, and if it can, how? 📚

This is an urgent question ⏳. Thank you, Jacob (a big fan and user of GitBook) 🚀📖🧑‍✈️🙌

**Comment by aikoii** [comment]
Hey @yankikahn! Appreciate your support for GitBook 💙!
Typically, we keep our watermark to help maintain affordable pricing and promote our brand. However, for our Enterprise plan customers, removing it is something we can look into.

**Comment by taszap** [comment]
Closing as a duplicate of #294
